guides/where-to-put-your-data.md

```diff
@@ -65,14 +65,17 @@ team to see what, if any shared Box folder you might have.
 4. Click on the "gear" icon by "Shared Link" (or the gear might be "Link Settings" depending on your browser)
 5. Make sure "Allow Download" is checked
 6. You should see a "Direct Link" field at the bottom of the panel. This is the
-   link to the file. 
+   link to the file. An intuitive URL can be obtained using the instructions below.
 
-If you need the as-uploaded filename you’ll need to look in the
-"Content-Disposition" header of the response. In the future there may be a URL
-redirection service to make the links more human-readable, but at the time of
-this writing that service is not yet available.
 
+#### Human-Friendly URLs to public Box files
+DMD maintains a redirector API to retrieve files from Box using a more intuitive URL than the default. The link is constructed as follows:
 
+1. The base of the URL is https://data.science.stsci.edu/redirect
```

Comment:
This doesn't need to hold up this PR, as it would require a change on the server-side... But I think `/redirect` is a but off-putting because the user shouldn't have to/want to know that there's any kind of redirection. Ideally this would instead be either just `https://data.science.stsci.edu/JWST/...` or possibly `/staticdata` or something like that?

Reply:
We want to keep the base route of the api gateway open for other potential uses. `/redirect` didn't have much thought, but when we later thought about it, we didn't come up with anything better. Could you make a ticket in the JUSI Jira instance if you do want us to change it?